.travis/yapf.sh

```diff
@@ -9,15 +9,24 @@ builtin cd "$(dirname "${BASH_SOURCE:-$0}")"
 ROOT="$(git rev-parse --show-toplevel)"
 builtin cd "$ROOT"
 
-lint() {
+format() {
     yapf \
         --style "$ROOT/.style.yapf" \
         --in-place --recursive --parallel \
         -- \
         "$@"
 }
 
-lint_all() {
+format_changed() {
+    # Formats python files that changed in last commit
+    git diff --name-only HEAD~1 HEAD | grep '\.py$' | xargs -P 5 \
```

Comment:
This isn't quite right. Usually you want to diff from the last `master` commit, not the one previous.